docs/taiko_in_docker.md

```diff
@@ -2,42 +2,74 @@
 layout: page.njk
 ---
 
-To run Taiko in a docker container follow these instructions.
+## Running Taiko in docker
 
-## Installing
+Sample code used in this documentation is available on Github. 
 
+The tutorial shows how to run Gauge and Taiko tests using Taiko. The docker image created as a part of this tutorial can be used to run tests locally and well or on a CI/CD setup.
 
-    docker pull node
+## Prerequisites
 
-Start a container
+* [NodeJS](https://nodejs.org/en/)
+* [Docker](https://www.docker.com/)
 
-    docker run -it node /bin/bash
+## Setup a sample project
 
-Update packages
+Create a Gauge+Taiko project. This article (and the sample) uses the default Gauge JavaScript template:
 
-    apt-get update
+```
+gauge init js
+```
 
-Install chromium dependencies
+## Create a Docker image
 
-    apt-get install -y wget unzip fontconfig locales gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils
+Use the [docker file](https://docs.docker.com/engine/reference/builder/) listed below to create a reusable docker image.
 
-Install Taiko
+## Dockerfile
 
-    npm install taiko -g --allow-root --unsafe-perm=true
+The project created using the command `gauge init js` has a [sample docker file](https://github.com/getgauge/template-js/blob/master/Dockerfile)
 
-Note: These instructions should ideally be a part of the `Dockerfile` while
-building a docker image.
+## Build the docker image
 
-## Running
+```
+docker build -t gauge-taiko .
+```
 
-To launch taiko in headless mode start taiko REPL with taiko and open browser with below arguments:
+## Run tests 
+```
+docker run --rm -v gauge-reports:/gauge/reports gauge-taiko
+```
 
-    await openBrowser({headless: true, args: ['--no-sandbox']});
+The reports will be available on the host machine in the gauge-reports directory after the run.
 
-To launch taiko in headful mode install Xvfb
+## View Reports
```

Comment:
Delete this section as it's already mentioned in the previous para where the report are generated

Reply:
done